docs/sources/flow/reference/components/prometheus.exporter.self.md

```diff
@@ -1,57 +1,57 @@
 ---
 aliases:
-- /docs/grafana-cloud/monitor-infrastructure/agent/flow/reference/components/prometheus.exporter.agent/
-- /docs/grafana-cloud/send-data/agent/flow/reference/components/prometheus.exporter.agent/
-canonical: https://grafana.com/docs/agent/latest/flow/reference/components/prometheus.exporter.agent/
+- /docs/grafana-cloud/monitor-infrastructure/agent/flow/reference/components/prometheus.exporter.self/
+- /docs/grafana-cloud/send-data/agent/flow/reference/components/prometheus.exporter.self/
```

Comment:
I'm not sure we need to update these aliases.  According to: https://grafana.com/docs/writers-toolkit/write/front-matter/#aliases the alias should be the original URL not the new URL.